src/_sass/core/_variables.scss

```diff
@@ -2,9 +2,13 @@
 // Colors
 $flutter-color-teal: #60caf6;
```

Comment:
These colors should match [those provided for Flutter](https://github.com/flutter/website/blob/main/src/_sass/base/_variables.scss#L2-L19). This would be consistent with the Dart design guidelines and Design's latest advice for the Flutter website.

Reply:
I can update this teal, but the other ones added come from the brand/design guidelines. Any in particular you're worried about?